api/csharp/pubsub-publish/src/pubsub-publish.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AssemblyName>pubsubpublish</AssemblyName>
+    <RootNamespace>toit.demos.api.csharp.pubsubpublish</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.13.0" />
+    <PackageReference Include="Grpc.Net.ClientFactory" Version="2.32.0" />
+    <PackageReference Include="Grpc.Tools" Version="2.32.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Protobuf Include="..\..\..\..\..\toitware\proto\toit\api\pubsub\publish.proto" GrpcServices="Client">
```

Comment:
Use the subdmodule in `third_party/toit_api` for this path.
Something like: `..\..\..\..\third_party\toit_api\proto\toit\api\pubsub\publish.proto`

Reply:
Using `\` in paths seems like a windows thing. Do you know if this will work on Mac/Linux as well?

Reply:
Done